testing/trino-product-tests-launcher/bin/run-launcher

```diff
@@ -5,7 +5,8 @@ target="${BASH_SOURCE%/*/*}/target"
 if command -v mvnd >/dev/null; then
     # Using mvnd here can save more than 2 seconds compared to ./mvnw startup cost. This is especially important
     # when invoking inspection commands like --help, env list, etc.
-    trino_version=$(mvnd -B help:evaluate -Dexpression=pom.version -q -DforceStdout --raw-streams)
```

Comment:
typo in commit message (whren)

Reply:
thanks, fixed.

also added a link to https://github.com/apache/maven-mvnd/issues/604